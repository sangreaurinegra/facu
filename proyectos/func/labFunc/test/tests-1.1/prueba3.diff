4d3
<   merge(low,mid,high);
6d4
< }
7a6
> {
8a8
>  h=low;
47,48d46
< cout<<"*******************************************************************
< *************"<<endl;
