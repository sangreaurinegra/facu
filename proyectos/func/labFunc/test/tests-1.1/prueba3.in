d
q
q
dp
dn
1d
0d
5,6d
9,4d
4,9d
d
q
q
+4-5+7d
.-2d
50d
;d
dasfasdfasdfasd
d2d1ds1321 12!
10 d
q
q
w
q
