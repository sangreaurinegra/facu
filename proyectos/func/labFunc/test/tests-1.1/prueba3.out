
#include <iostream.h>
int a[50];
 }
void merge(int low,int mid,int high)
{
 int h,i,j,b[50],k;
 h=low;
 i=low;

 while((h<=mid)&&(j<=high))
 {
  if(a[h]<=a[j])
  {
   b[i]=a[h];
   h++;
  }
  else
  {
   b[i]=a[j];
   j++;
  }
  i++;
 }
 if(h>mid)
 {
  for(k=j;k<=high;k++)
  {
   b[i]=a[k];
   i++;
  }
 }
 else
 {
  for(k=h;k<=mid;k++)
  {
   b[i]=a[k];
   i++;
  }
 }
 for(k=low;k<=high;k++) a[k]=b[k];
}
void main()
{
 int num,i;

